bedrock/pocket/urls.py

```diff
@@ -5,21 +5,90 @@
 from bedrock.mozorg.util import page
 
 urlpatterns = (
-    page("about/", "pocket/about.html"),
-    page("add/", "pocket/add.html"),
-    page("android/", "pocket/android.html"),
-    page("ios/", "pocket/ios.html"),
-    page("chrome/", "pocket/chrome.html"),
-    page("safari/", "pocket/safari.html"),
-    page("opera/", "pocket/opera.html"),
-    page("edge/", "pocket/edge.html"),
-    page("welcome/", "pocket/welcome.html"),
-    page("contact-info/", "pocket/contact-info.html"),
-    page("firefox/new_tab_learn_more/", "pocket/firefox/new-tab-learn-more.html"),
-    page("pocket-and-firefox/", "pocket/pocket-and-firefox.html"),
-    page("get-inspired/", "pocket/get-inspired.html"),
-    page("jobs/", "pocket/jobs.html"),
-    page("privacy/", "pocket/privacy.html"),
-    page("tos/", "pocket/tos.html"),
-    page("save-to-pocket/", "pocket/save-to-pocket.html"),
+    page(
+        "about/",
+        "pocket/about.html",
+        url_name="pocket.about",
```

Comment:
These `url_name` args are superfluous I believe. The default name of the URL will be the path to the template with `/` replaced with `.` and with the `.html` stripped off.

Reply:
Note: Fine with me to leave like this, but just FYI